nuxt.config.js

```diff
@@ -58,7 +58,6 @@ export default {
         /*
          ** You can extend webpack config here
          */
-        extend(config, ctx) {},
```

Comment:
What was the rational for removing this? I might not understand Vue well enough but I couldn't see how it relates to the other changes here.

Reply:
Oh it just want being used or doing anything. I think it was there as a default when you set the project up.

Reply:
Ok. Sounds good!